.github/external_ci/appveyor.yml

```diff
@@ -1,5 +1,19 @@
 version: '{build}'
 
+# only build PRs and commits to develop branch
+# (see https://help.appveyor.com/discussions/questions/55079-two-builds-per-commit-to-pull-request)
+branches:
+  only:
+    - develop
+
+only_commits:
```

Comment:
I gave this another thought: I do not think that we should be "too clever" about when to trigger a CI job. There may always be edge cases where a file outside this list can break a build. I get that we don't want to fire up dozens of Docker images for typo fixes, but at the same time, I would want to state with confidence "this does not break anyone's build".

Reply:
Note that this only affects AppVeyor, though. The vast majority of tests are still run.

Reply:
I see. Let's hope this doesn't bite us in the future.